modules/replicator/src/integrationTest/java/org/apache/ignite/internal/replicator/ItPlacementDriverReplicaSideTest.java

```diff
@@ -257,13 +259,19 @@ public void beforeTest(TestInfo testInfo) {
                     () -> IgniteUtils.shutdownAndAwaitTermination(partitionOperationsExecutor, 10, TimeUnit.SECONDS)
             ));
         }
+
+        grpNodes = chooseRandomNodes(3);
```

Comment:
Why do we need to move chooseRandomNodes(3); from tests itself to beforeTest?

Reply:
I did it because to `@AfterEach` we must to move `grpNodes` as a field. Then, there are options: left `chooseRandomNodes` in each of actually 2 tests (with the same argument) or move it to `@BeforeEach` that looks like more clean (no code duplication) and idiomatic (starting before and stopping after no mater what happened in tests).

Reply:
Ok.